webapp/src/components/node.tsx

```diff
@@ -34,7 +34,7 @@ const Node = (props: Props) => {
         const videoItems = node.videos? node.videos.map(video => {
             return {
                 areaLabel: video.name,
-                display_name: video.name + " (" + Math.floor(video.length/60) + " min)",
+                display_name: video.name + " (" + (Math.floor(video.length/60) + 1) + " min)",
```

Comment:
Would it be better to move this to a separate function? The same logic is used in `rhs.tsx`